clients/src/main/java/org/apache/kafka/common/errors/RecordDeserializationException.java

```diff
@@ -19,7 +19,8 @@
 import org.apache.kafka.common.TopicPartition;
 
 /**
- *  Any exception encountered on an invalid record (deserialization error, corrupt record, etc...)
+ *  This exception is raised for any error that occurs while deserializing records received by the consumer using 
+ *  the configured {@link Deserializer}.
```

Comment:
You need the fully qualified class name:
```java
{@link org.apache.kafka.common.serialization.Deserializer}
```

Reply:
done